MyOwnORM/MyOwnORM/Repository/Interface/ICustomDbSetRepository.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM.Repository
+{
+    public interface ICustomDbSetRepository<T> where T : class
```

Comment:
methods should be async here and in implementation

Reply:
Done